src/main/java/com/sourcery/gymapp/backend/authentication/controller/PermissionController.java

```diff
@@ -0,0 +1,50 @@
+package com.sourcery.gymapp.backend.authentication.controller;
+
+import com.sourcery.gymapp.backend.authentication.dto.CreatePermissionDto;
+import com.sourcery.gymapp.backend.authentication.dto.PermissionResponseDto;
+import com.sourcery.gymapp.backend.authentication.model.Permission;
+import com.sourcery.gymapp.backend.authentication.service.PermissionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/permissions")
+@RequiredArgsConstructor
+public class PermissionController {
```

Comment:
there have to be a permissions to do crud operations with this controller.
Cause we don't want an average user to accidentally do any CRUD with these